rtcp-packet_test.go

```diff
@@ -0,0 +1,157 @@
+package rtp
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+)
+
+
+func unreachable(t *testing.T) {
+  t.Fatalf("unreachable reached")
+}
+
+func assertEqual(t *testing.T, a interface{}, b interface{}) {
+	if a != b {
+    println("actual: ", a)
+    println("expected: ", b)
+		t.Fatalf("%s != %s", a, b)
+	}
+}
+
+func compareByteArrays(t *testing.T, actual []byte, expected []byte) {
```

Comment:
If you're going to define a utility method, please use it to replace other uses of `bytes.Equal`.  And then move it to a test utility file, say `util_test.go`.  Same for `assertEqual`.